packages/mobile-first-router/src/MFNavigationTypes.d.ts

```diff
@@ -30,3 +30,14 @@ export interface MFNavigationConfig {
   width?: number;
   height?: number;
 }
+
+export interface MFNavigationReducerConfig {
+  initialTabRoutes?: string[];
+  routeConfig: MFNavigationConfig;
+  adapter?: {
+    getRoute: Function;
+    setRoute: Function;
+    getQueryString: Function;
+    getUrlState: Function;
```

Comment:
anyway we can make this work with just getRoute and setRoute?

Reply:
I could extract the other two and make them functions inside the reducer, I guess?

Reply:
Actually I think this should be good for now

Reply:
Maybe make a function called getTab?

Reply:
Whenever they SET_ACTIVE_TAB, we can change the URL to reflect it

Reply:
just like set route really